### DISCUSSION TITLE
How to use Tailwind class sorter?

### DISCUSSION THREAD

**Discussion by adamzerner** [freshframework/fresh#2284] [Q&A] [answered]
I read https://tailwindcss.com/blog/automatic-class-sorting-with-prettier the other day and would love to be able to utilize it in my [Fresh project](https://github.com/adamzerner/rfui). However, I'm not sure how to get set up.

It says that you need to 1) install the NPM package and 2) update your Prettier config to include the plugin. Step 1 is doable enough with NPM specifiers. Step 2 is the part that I'm confused about. As a Deno project, we use `deno fmt` not Prettier. The former has a [configuration file](https://docs.deno.com/runtime/manual/getting_started/configuration_file#fmt), but nothing related to plugins.

It seems like it's probably not possible to do without actually using Prettier. Which sucks because using Prettier instead of `deno fmt` seems pretty against the idea of how Deno comes with stuff "for free".

I also am interested in hearing if people have alternative solutions to this problem that they like.

**Comment by nnmrts** [comment, marked as answer]
@adamzerner I can recommend you this extension in case you use vscode: https://github.com/Trapfether/tailwind-raw-reorder

It's a maintained fork of https://github.com/heybourn/headwind, which I used before but is now unmaintained. It works for me with the latest tailwind version in fresh jsx files.

**Reply by adamzerner** [reply]
Works like a charm. Thanks!
